waku/v2/node/config.nim

```diff
@@ -31,7 +31,12 @@ type
       desc: "TCP listening port."
       defaultValue: 60000
       name: "tcp-port" }: Port
-
+    
+    wsPort* {.
+      desc: "Web Socket listening port."
+      defaultValue: 1400
```

Comment:
This is an odd value.

It's an http port. Commonly used value for http server run by a user (ie, port > 1024) are:
- 8080
- 8101
- 8000
- Something that looks like 80, the default http port.